README.md

```diff
@@ -10,3 +10,11 @@ All `obal` actions should also work with plain Ansible when called like `ansible
 
 - `python` (2 or 3)
 - `ansible`
+
+## Using Obal via Container
+
+Obal and all it's required packages are available in a container that can be used locally or in build environments. The users Koji credentials and configuration must be mounted into the container alongside of mounting the packaging project into `/opt/packaging` to work. Note the examples below assume SELinux is disabled.
+
+To run:
+
+    podman run -v `pwd`:/opt/packaging -v ~/.koji:/root/.koji obal:latest scratch katello
```

Comment:
It would be good to mention that podman is a drop in for docker. I didn't know it was a drop in.